test_fsrefs.py failures on aarch64 alternative/tiny

When testing #542 I got errors on these vmtest kernels:
```
Failed: aarch64: 
5.14.21-vmtest36.0alternative,
5.13.19-vmtest36.0alternative, 
5.12.19-vmtest36.0alternative, 
5.11.22-vmtest36.0alternative, 
5.10.244-vmtest36.0alternative, 
5.4.299-vmtest36.0.1alternative, 
4.19.325-vmtest36.0.1alternative, 

5.11.22-vmtest36.0tiny, 
5.10.244-vmtest36.0tiny,
5.4.299-vmtest36.0.1tiny, 
4.19.325-vmtest36.0.1tiny
```

The failures:

```
__________________________________________ TestFsRefs.test_btrfs_subvolume __________________________________________

self = <tests.linux_kernel.tools.test_fsrefs.TestFsRefs testMethod=test_btrfs_subvolume>

    @skip_unless_have_test_disk
    def test_btrfs_subvolume(self):
        disk = os.environ["DRGN_TEST_DISK"]
        with contextlib.ExitStack() as exit_stack:
            subprocess.check_call(["mkfs.btrfs", "-qf", disk])
    
>           mount(disk, self._tmp, "btrfs")

tests/linux_kernel/tools/test_fsrefs.py:242: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/linux_kernel/__init__.py:312: in mount
    _check_ctypes_syscall(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

ret = -1, args = ('/dev/vda', None, PosixPath('/tmp/tmp_ul0ebr6')), errno = 22

    def _check_ctypes_syscall(ret, *args):
        if ret == -1:
            errno = ctypes.get_errno()
>           raise OSError(errno, os.strerror(errno), *args)
E           OSError: [Errno 22] Invalid argument: '/dev/vda' -> PosixPath('/tmp/tmp_ul0ebr6')

tests/linux_kernel/__init__.py:307: OSError
----------------------------------------------- Captured stderr call ------------------------------------------------
WARNING: sectorsize 4096 does not match host CPU page size 65536, with kernels 6.x and up
         the 4KiB sectorsize is supported on all architectures but other combinations
         may fail the filesystem mount, use "--sectorsize 65536" to override that

____________________________________ TestFsRefs.test_super_block_on_block_device ____________________________________

self = <tests.linux_kernel.tools.test_fsrefs.TestFsRefs testMethod=test_super_block_on_block_device>

    @skip_unless_have_test_disk
    def test_super_block_on_block_device(self):
        disk = os.environ["DRGN_TEST_DISK"]
        for fstype, mkfs in (
            ("ext2", ("mke2fs", "-qF")),
            ("btrfs", ("mkfs.btrfs", "-qf")),
        ):
            with self.subTest(fstype=fstype):
                subprocess.check_call([*mkfs, disk])
    
                with contextlib.ExitStack() as exit_stack:
>                   mount(disk, self._tmp, fstype)

tests/linux_kernel/tools/test_fsrefs.py:222: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/linux_kernel/__init__.py:312: in mount
    _check_ctypes_syscall(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

ret = -1, args = ('/dev/vda', None, PosixPath('/tmp/tmpn4ebcjg3')), errno = 22

    def _check_ctypes_syscall(ret, *args):
        if ret == -1:
            errno = ctypes.get_errno()
>           raise OSError(errno, os.strerror(errno), *args)
E           OSError: [Errno 22] Invalid argument: '/dev/vda' -> PosixPath('/tmp/tmpn4ebcjg3')

tests/linux_kernel/__init__.py:307: OSError
----------------------------------------------- Captured stderr call ------------------------------------------------
WARNING: sectorsize 4096 does not match host CPU page size 65536, with kernels 6.x and up
         the 4KiB sectorsize is supported on all architectures but other combinations
         may fail the filesystem mount, use "--sectorsize 65536" to override that

```

During the test runs, I can see the following messages (which I think are printk messages):
```
tests/linux_kernel/tools/test_fsrefs.py::TestFsRefs::test_binfmt_misc_in_user_ns SKIPPED (kernel does not...) [ 96%]
tests/linux_kernel/tools/test_fsrefs.py::TestFsRefs::test_btrfs_subvolume BTRFS: device fsid 933b76d2-5e5d-4cae-a9e7-9859455c44d5 devid 1 transid 8 /dev/vda scanned by mkfs.btrfs (175)
BTRFS info (device vda): using free space tree
BTRFS info (device vda): has skinny extents
BTRFS error (device vda): subpage sectorsize 4096 only supported read-only for page size 65536
BTRFS error (device vda): open_ctree failed
FAILED                              [ 97%
```

--- a/tests/linux_kernel/tools/test_fsrefs.py
+++ b/tests/linux_kernel/tools/test_fsrefs.py
@@ -213,7 +213,7 @@ class TestFsRefs(LinuxKernelTestCase):
         disk = os.environ["DRGN_TEST_DISK"]
         for fstype, mkfs in (
             ("ext2", ("mke2fs", "-qF")),
-            ("btrfs", ("mkfs.btrfs", "-qf")),
+            ("btrfs", ("mkfs.btrfs", "-qf", "-s", str(mmap.PAGESIZE))),
         ):
             with self.subTest(fstype=fstype):
                 subprocess.check_call([*mkfs, disk])
@@ -237,7 +237,7 @@ class TestFsRefs(LinuxKernelTestCase):
     def test_btrfs_subvolume(self):
         disk = os.environ["DRGN_TEST_DISK"]
         with contextlib.ExitStack() as exit_stack:
-            subprocess.check_call(["mkfs.btrfs", "-qf", disk])
+            subprocess.check_call(["mkfs.btrfs", "-qf", "-s", str(mmap.PAGESIZE), disk])
 
             mount(disk, self._tmp, "btrfs")
             exit_stack.callback(umount, self._tmp)

